build/commands/lib/test.js

```diff
@@ -29,42 +29,20 @@ const getTestBinary = (config, suite) => {
   return path.join(config.outputDir, testBinary)
 }
 
-const getChromiumUnitTestsSuites = () => {
-  return [
-    'base_unittests',
-    'components_unittests',
-    'content_unittests',
-    'net_unittests',
-    'unit_tests',
-  ]
-}
-
-const getBraveUnitTestsSuites = (config) => {
-  let tests = []
-  if (config.isIOS()) {
-    tests.push('ios_brave_unit_tests')
-  } else {
-    tests.push('brave_unit_tests')
-  }
-  tests.push('brave_components_unittests')
+const getTestsToRun = (config, suite) => {
+  let suiteNames = []
 
-  if (!config.isMobile()) {
-    tests.push('brave_installer_unittests')
+  const testDepFile = path.join(config.outputDir, `${suite}.json`)
+  if (fs.existsSync(testDepFile)) {
+    suiteDepNames = JSON.parse(
+      fs.readFileSync(testDepFile, { encoding: 'utf-8' }),
+    )
+    for (const dep of suiteDepNames) {
+      suiteNames.push(dep.split(':').pop())
```

Comment:
alternative: dep.split(':').at(-1)

Reply:
alternative:
```
suiteNames = suiteDepNames.map(x=> x.split(':').at(-1))
```

Reply:
it's a pretty common pattern we use in other places, but I also don't have an issue with using `at`